python/ola/PidStore.py

```diff
@@ -160,6 +160,25 @@ def ValidateAddressing(self, args, command_class):
   def __cmp__(self, other):
     return cmp(self._value, other._value)
 
+  def __eq__(self, other):
+    return self.value == other.value \
```

Comment:
I think we favour behaving like C++, putting the and on the previous lines and aligning the self.values.

Reply:
Can we align the `self.` above each other, or does flake8 complain?

Reply:
No we can't.